Owner.rb

```diff
@@ -0,0 +1,74 @@
+require "./bank.rb"
+require "csv"
+module Bank
+
+  class Owner
+
+     attr_accessor :id, :last_name, :first_name, :address, :city, :state
+
+     def initialize (info_hash)
+       @id = info_hash [:id]
+       @last_name = info_hash [:last_name]
+       @first_name = info_hash [:first_name]
+       @address = info_hash [:address]
+       @city = info_hash [:city]
+       @state = info_hash [:state]
+     end
+
+     def self.create_owner (person_array)
+       person_id = person_array[0].to_i
+       person_last_name = person_array[1]
+       person_first_name = person_array[2]
+       person_address = person_array[3]
+       person_city = person_array[4]
+       person_state = person_array[5]
+       owner =   {id: person_id,
+                 last_name: person_last_name,
```

Comment:
Since you're not re-using these local variables anywhere else, it might make sense to remove the local variables and plug the `person_array[1]